### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. basic modal
BasicModal
2. modal with stacked button
StackedModal

### CODE REVIEW

File: css/components/modal.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* === css for modal ==== */
+
+/* == basic modal == */
+.modal{
+    box-shadow: 0px 3px 10px rgba(0,0,0,0.4);
```
Comment (tanishq20) [inline css/components/modal.css]: You can you var() for colors.

**Review by tanishq20 — COMMENTED** [review]
Very clean code. Suggested small change.

File: css/components/modal.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/components/modal.css]: sure i have only used this for box shadow. That's why i have explicitly added this

**Review by tanishq20 — APPROVED** [review]

File: docs/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        <div class="docs-wrapper">
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <img src="../../assets/brand-logo.svg" alt="Logo">
+                        <h3>Pegasus</h3>
+                    </div>
+                    <nav class="links">
+                        <ul>
+                           <li><a href="https://github.com/RahulSingh9131">Github</a></li>
```
Comment (shubhamkumar648) [inline docs/modal/modal.html]: add blank attributes so it is open on new tab

File: docs/modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                           <li><a href="https://twitter.com/singhrahul3112">Twitter</a></li>
```
Comment (shubhamkumar648) [inline docs/modal/modal.html]: same  in all social icon

**Review by shubhamkumar648 — COMMENTED** [review]
consider my change

File: docs/modal/modal.html (hunk 3 above)
Comment (shubhamkumar648) [reply docs/modal/modal.html]: perfect

**Review by shubhamkumar648 — APPROVED** [review]
you can merge

**Pull request merged**